services/bots/src/github-webhook/handlers/validate-cla.ts

```diff
@@ -0,0 +1,269 @@
+import { ConfigService } from '@nestjs/config';
+import { BaseWebhookHandler } from './base';
+
+import { DynamoDB } from 'aws-sdk';
+
+const ignoredAuthors: Set<string> = new Set([
+  // Ignore bot accounts that are not masked as bots
+  'travis@travis-ci.org',
+  'ImgBotHelp@gmail.com',
+  'support@lokalise.com',
+]);
+
+const ignoredRepositories: Set<string> = new Set([
+  // Ignore repositories that does not contain code
+  'home-assistant/home-assistant.io',
+  'home-assistant/assets',
+]);
+
+export class ValidateCla extends BaseWebhookHandler {
+  private ddbClient: DynamoDB;
+
+  constructor(configService: ConfigService) {
+    super(configService);
+    this.ddbClient = new DynamoDB({ region: 'us-west-2' });
+  }
+
+  async handle(eventType: string, eventData: Record<string, any>) {
+    if (!eventType.startsWith('pull_request')) {
+      return;
+    }
+
+    if (ignoredRepositories.has(eventData.repository.full_name)) {
+      return;
+    }
+
+    if (!['synchronize', 'opened', 'labeled'].includes(eventData.action)) {
+      return;
+    }
+    if (eventData.action === 'labeled') {
+      if (eventData.label.name !== 'cla-recheck') {
+        return;
+      } else if (eventData.label.name === 'cla-recheck') {
```

Comment:
you can remove this else-statement because that's exactly what the `if` check filters.